File: SocialProWeb/src/SocialPro/UserBundle/Entity/Trajet.php
<?php

namespace SocialPro\UserBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Trajet
 *
 * @ORM\Table(name="trajet", indexes={@ORM\Index(name="idUser", columns={"id_user_trajet"})})
 * @ORM\Entity(repositoryClass="SocialPro\UserBundle\Repository\TrajetRepository")
 */
class Trajet
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id_trajet", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idTrajet;

    /**
     * @var string
     *
     * @ORM\Column(name="depart", type="text", length=65535, nullable=false)
     */
    private $depart;

    /**
     * @var string
     *
     * @ORM\Column(name="arrive", type="text", length=65535, nullable=false)
     */
    private $arrive;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date", type="date", nullable=false)
     */
    private $date;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="heure", type="time", nullable=false)
     */
    private $heure;

    /**
     * @var integer
     *
     * @ORM\Column(name="nombre_place", type="integer", nullable=false)
     */
    private $nombrePlace;

    /**
     * @var \User
     *
     * @ORM\ManyToOne(targetEntity="User")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_user_trajet", referencedColumnName="id")
     * })
     */
    private $idUserTrajet;



    /**
     * Get idTrajet
     *
     * @return integer
     */
    public function getIdTrajet()
    {
        return $this->idTrajet;
    }

    /**
     * Set depart
     *
     * @param string $depart
     *
     * @return Trajet
     */
    public function setDepart($depart)
    {
        $this->depart = $depart;

        return $this;
    }

    /**
     * Get depart
     *
     * @return string
     */
    public function getDepart()
    {
        return $this->depart;
    }

    /**
     * Set arrive
     *
     * @param string $arrive
     *
     * @return Trajet
     */
    public function setArrive($arrive)
    {
        $this->arrive = $arrive;

        return $this;
    }

    /**
     * Get arrive
     *
     * @return string
     */
    public function getArrive()
    {
        return $this->arrive;
    }

    /**
     * Set date
     *
     * @param \DateTime $date
     *
     * @return Trajet
     */
    public function setDate($date)
    {
        $this->date = $date;

        return $this;
    }

    /**
     * Get date
     *
     * @return \DateTime
     */
    public function getDate()
    {
        return $this->date;
    }

    /**
     * Set heure
     *
     * @param \DateTime $heure
     *
     * @return Trajet
     */
    public function setHeure($heure)
    {
        $this->heure = $heure;

        return $this;
    }

    /**
     * Get heure
     *
     * @return \DateTime
     */
    public function getHeure()
    {
        return $this->heure;
    }

    /**
     * Set nombrePlace
     *
     * @param integer $nombrePlace
     *
     * @return Trajet
     */
    public function setNombrePlace($nombrePlace)
    {
        $this->nombrePlace = $nombrePlace;

        return $this;
    }

    /**
     * Get nombrePlace
     *
     * @return integer
     */
    public function getNombrePlace()
    {
        return $this->nombrePlace;
    }

    /**
     * Set idUserTrajet
     *
     * @param \SocialPro\UserBundle\Entity\User $idUserTrajet
     *
     * @return Trajet
     */
    public function setIdUserTrajet(\SocialPro\UserBundle\Entity\User $idUserTrajet = null)
    {
        $this->idUserTrajet = $idUserTrajet;

        return $this;
    }

    /**
     * Get idUserTrajet
     *
     * @return \SocialPro\UserBundle\Entity\User
     */
    public function getIdUserTrajet()
    {
        return $this->idUserTrajet;
    }
}

File: SocialProMobile/src/Entity/Trajet.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entity;



/**
 *
 * @author Choura
 */
public class Trajet {
    
    private int idTrajet;
    private int idUserTrajet;
    private String depart;
    private String arrive;
    private String date;
    private String heure;
    private int nombrePlace;

    public Trajet() {
    }

    public Trajet(int idTrajet, int idUserTrajet, String depart, String arrive, String date, String heure, int nombrePlace) {
        this.idTrajet = idTrajet;
        this.idUserTrajet = idUserTrajet;
        this.depart = depart;
        this.arrive = arrive;
        this.date = date;
        this.heure = heure;
        this.nombrePlace = nombrePlace;
    }

    public int getIdTrajet() {
        return idTrajet;
    }

    public void setIdTrajet(int idTrajet) {
        this.idTrajet = idTrajet;
    }

    public int getIdUserTrajet() {
        return idUserTrajet;
    }

    public void setIdUserTrajet(int idUserTrajet) {
        this.idUserTrajet = idUserTrajet;
    }

    public String getDepart() {
        return depart;
    }

    public void setDepart(String depart) {
        this.depart = depart;
    }

    public String getArrive() {
        return arrive;
    }

    public void setArrive(String arrive) {
        this.arrive = arrive;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getHeure() {
        return heure;
    }

    public void setHeure(String heure) {
        this.heure = heure;
    }

    public int getNombrePlace() {
        return nombrePlace;
    }

    public void setNombrePlace(int nombrePlace) {
        this.nombrePlace = nombrePlace;
    }

    @Override
    public int hashCode() {
        int hash = 7;
        hash = 67 * hash + this.idTrajet;
        return hash;
    }

    @Override
    public boolean equals(Object obj) {
        if (this == obj) {
            return true;
        }
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final Trajet other = (Trajet) obj;
        if (this.idTrajet != other.idTrajet) {
            return false;
        }
        return true;
    }

    @Override
    public String toString() {
        return "Trajet{" + "idTrajet=" + idTrajet + ", idUserTrajet=" + idUserTrajet + ", depart=" + depart + ", arrive=" + arrive + ", date=" + date + ", heure=" + heure + ", nombrePlace=" + nombrePlace + '}';
    }   
    
}
